controllers/platform/platformutils.go

```diff
@@ -101,27 +103,27 @@ func SetPlatformDefaults(p *operatorapi.SonataFlowPlatform, verbose bool) error
 		defaultKanikoBuildCache := "false"
 		p.Spec.BuildPlatform.BuildStrategyOptions[kanikoBuildCacheEnabled] = defaultKanikoBuildCache
 		if verbose {
-			log.Log.Infof("Kaniko cache set to %s", defaultKanikoBuildCache)
+			klog.V(log.I).InfoS("Kaniko cache set", "value", defaultKanikoBuildCache)
 		}
 	}
 
 	setStatusAdditionalInfo(p)
 
 	if verbose {
-		log.Log.Infof("BaseImage set to %s", p.Spec.BuildPlatform.BaseImage)
-		log.Log.Infof("Timeout set to %s", p.Spec.BuildPlatform.GetTimeout())
+		klog.V(log.I).InfoS("BaseImage set", "value", p.Spec.BuildPlatform.BaseImage)
+		klog.V(log.I).InfoS("Timeout set", "value", p.Spec.BuildPlatform.GetTimeout())
 	}
 	return nil
 }
 
 func setStatusAdditionalInfo(platform *operatorapi.SonataFlowPlatform) {
 	platform.Status.Info = make(map[string]string)
 
-	log.Debugf("SonataFlow Platform [%s]: setting build publish strategy", platform.Namespace)
+	klog.V(log.D).InfoS("SonataFlow Platform: setting build publish strategy", "namespace", platform.Namespace)
```

Comment:
Infof to have the placeholder "SonataFlow Platform [namespace]:"

Reply:
@ricardozanini wanted to introduce structured logging.

Reply:
@manstis ok, but the phrase "SonataFlow Platform: setting build publish strategy" means nothing, almost remove the ":"